velox/flag_definitions/flags.h

```diff
@@ -17,3 +17,9 @@
 namespace facebook::velox {
 void translateFlagsToGlobalConfig();
 }
+
+/// Use O_DIRECT for SSD cache IO
+DECLARE_bool(velox_ssd_odirect);
+
+/// Read back data after writing to SSD
```

Comment:
@majetideepak Would you expand the comment to explain a bit more. Why would we sometimes read data and some time don't?